src/main/java/woohakdong/server/domain/admin/adminAccountHistory/AdminAccountHistory.java

```diff
@@ -0,0 +1,53 @@
+package woohakdong.server.domain.admin.adminAccountHistory;
+
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import woohakdong.server.domain.admin.adminAccount.AccountType;
+import woohakdong.server.domain.admin.adminAccount.AdminAccount;
+import woohakdong.server.domain.clubAccount.ClubAccount;
+
+import java.time.LocalDate;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+public class AdminAccountHistory {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long adminAccountHistoryId;
+
+    @Column(nullable = false)
+    @Enumerated(EnumType.STRING)
+    private AccountType adminAccountHistoryInOutType;
+
+    @Column(nullable = false)
+    private LocalDate adminAccountHistoryTranDate;
+
+    @Column(nullable = false)
+    private Long adminAccountHistoryBalanceAmt;
+
+    @Column(nullable = false)
+    private Long adminAccountHistoryTranAmt;
```

Comment:
```suggestion
    @Column(nullable = false)
    private Long adminAccountHistoryBalanceAmount;

    @Column(nullable = false)
    private Long adminAccountHistoryTranAmount;
```

다른 table들 column명이랑 맞춰야해서 이렇게 수정하면 좋을 것 같습니다